### DISCUSSION TITLE
Welcome to VulkanSceneGraph Discussions!

### DISCUSSION THREAD

**Discussion by vsg-dev** [vsg-dev/VulkanSceneGraph#546] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robertosfield** [comment]
VulkanSceneGraph Discussions forum is replacing the [vsg-uses](https://groups.google.com/g/vsg-users/pending-members) google group set up early in the project life.  We will keep the vsg-users list around for existing subscribers and as an archive for the various discussions that we've had since it's inception in November 2018.

I would like existing subscribers to the vsg-users googlegroup to start using this github hosted VulkanSceneGraph Discussions forum, we may have a few teething problems, if you come across issues feel free to use the vsg-users googlegroup.  Once this new Discussion forum is running smoothly we can look to stopping new posts to vsg-users and just use it as a historical archive,

**Reply by kannode** [reply]
Robert, I believe discussions is a better option as we can hashtag directly to code if necessary and all in one place.
